llvm/test/CodeGen/SPIRV/function-alias-non-interposable.ll

```diff
@@ -0,0 +1,22 @@
+; RUN: llc -O0 -mtriple=spirv64-unknown-unknown %s -o - | FileCheck %s
```

Comment:
A tip, if you feed `%s` through `stdin` it automatically sends the output to stdout.

```suggestion
; RUN: llc -O0 -mtriple=spirv64-unknown-unknown < %s | FileCheck %s
```